apps/frontend/src/components/questionary/questionaryComponents/RichTextInput/QuestionRichTextInputForm.tsx

```diff
@@ -27,6 +28,7 @@ export const QuestionRichTextInputForm = (props: QuestionFormProps) => {
     >
       {(formikProps) => (
         <>
+          <TemplateEditLabel pageType="Question" />
```

Comment:
Instead of repeating the same label component in all of the forms I was thinking if it is possible to take it out of the form and make it global. Something like putting it in the https://github.com/UserOfficeProject/user-office-core/blob/develop/apps/frontend/src/components/template/QuestionTemplateRelationEditor.tsx before the form itself and that way will be global for all of the questions in one place. We can try the same for https://github.com/UserOfficeProject/user-office-core/blob/develop/apps/frontend/src/components/template/QuestionEditor.tsx